codecs/vlen-bytes/schema.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "anyOf": [
+    {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "enum": ["vlen-bytes"]
+        },
+        "configuration": {
+          "type": "object",
+          "additionalProperties": false
+        }
+      },
+      "required": ["name"],
+      "additionalProperties": false
+    },
+    { "type": "string", "enum": ["vlen-bytes"] }
```

Comment:
Since this normalization is part of the zarr spec itself, it is a bit unfortunate to duplicate it for everything.  On the other hand I guess there may be no other way to allow these schemas to actually be used for programmatic verification.

Reply:
I was thinking that we could add some codegen utils in this repo to generate these schema.json files. For example, the source could be either the object notation or string notation and the other would be automatically added, if applicable.